.changeset/silver-guests-brake.md

```diff
@@ -0,0 +1,5 @@
+---
+'microbundle': minor
```

Comment:
Do you consider this a breaking change?

Reply:
Used to considered it to be a fix + feature instead of break, and it can fallback to previous behavior.

After thought more carefully... Maybe you're right, in theroy if user previously used different TS version during developing & bundling, it will lead to different result.

But I'm still hesitating. Logically if a user previously:
* used same version in dev & bundling - no change
* verA in dev + verB in bundling, both worked - use verA in both stages should also work
* verA in dev + verB in bundling, OK in dev but failed to bundle - this PR means a fix i.o. a break to this user
* verA in dev + verB in bundling, OK in bundling but failed in dev - this PR is a break, but I'm not sure if this just happen...

Anyway, major is always a safer choice... Shall I update the PR?